parsing/dialects/defaultdialect/src/test/java/org/apache/calcite/test/DefaultDialectQuidemTest.java

```diff
@@ -76,4 +78,34 @@ public static void main(String[] args) throws Exception {
       }
     };
   }
+
+  /** Override settings for "sql/misc.iq". */
+  public void testSqlMisc(String path) throws Exception {
+    switch (CalciteAssert.DB) {
+    case ORACLE:
+      // There are formatting differences (e.g. "4.000" vs "4") when using
+      // Oracle as the JDBC data source.
+      return;
+    }
+    try (TryThreadLocal.Memo ignored = Prepare.THREAD_EXPAND.push(true)) {
+      checkRun(path);
+    }
+  }
+
+  /** Override settings for "sql/scalar.iq". */
+  public void testSqlScalar(String path) throws Exception {
+    try (TryThreadLocal.Memo ignored = Prepare.THREAD_EXPAND.push(true)) {
+      checkRun(path);
+    }
+  }
+
+  /** Runs the dummy script "sql/dummy.iq", which is checked in empty but
+   * which you may use as scratch space during development. */
+
+  // Do not disable this test; just remember not to commit changes to dummy.iq
+  public void testSqlDummy(String path) throws Exception {
+    try (TryThreadLocal.Memo ignored = Prepare.THREAD_EXPAND.push(true)) {
+      checkRun(path);
+    }
```

Comment:
This is a bit weird though I do see what it does. Would be possible to not commit this test at all after work is complete?

Reply:
Yeah it is kind of odd, I just copied it from the [CoreQuidemTest](https://github.com/googleinterns/calcite/blob/master/core/src/test/java/org/apache/calcite/test/CoreQuidemTest.java#L73). since some tests were also failing without the testSqlMisc method above. I tried removing testSqlDummy and testSqlScalar and the tests still pass so it can probably just be removed